### PR TITLE
Add bucket tests to test suite

### PR DESCRIPTION
**Pull request by Oozyx**
## Changes

- Refactor tests to run in a test suite

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Review by joshLong145 — APPROVED** [review]

File: internal/bucket/bucket_package_test.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		{
+			Name:      "test",
+			Uri:       suite.cid,
+			Timestamp: time.Now().Unix(),
+			Type:      bt.ResourceIdentifier_CID,
+		},
+	}
+
+	_, suite.testBucket, _ = suite.motorNode.CreateBucket(mt.CreateBucketRequest{
+		Creator:    suite.motorNode.GetAddress(),
```
Comment (ntindle) [inline internal/bucket/bucket_package_test.go:107]: This test suite for buckets should also test
- Adding items to existing bucket
- Removing items from existing bucket
- Modifying items in a bucket
- Listing items in a bucket

**Review by ntindle — CHANGES_REQUESTED** [review]

**Comment by ntindle** [comment]
Great job on getting the tests working

File: internal/bucket/bucket_package_test.go (hunk 1 above)
Comment (joshLong145) [reply internal/bucket/bucket_package_test.go:107]: to add a couple more
- adding a bucket to a bucket
- accessing a bucket from a bucket `resolveBuckets` will get all buckets at the top level of the single bucket this should be tested for respect to `bucketItems` that specify a bucket as a piece of content.

**Review by ntindle — APPROVED** [review]

**Comment by mcjcloud** [comment]
Is this ready to merge?

**Pull request merged**